lib/kits/core-ui/components/common/offer-list.ts

```diff
@@ -57,6 +59,7 @@ const OfferList = ({
         id?: string;
         businessId?: string;
     };
+    sgnData: any;
```

Comment:
Please put the actual type, this one is already well-known: https://github.com/tjek/tjek-js-sdk/blob/develop/lib/kits/core-ui/paged-publication.ts#L41